### DISCUSSION TITLE
How to pass environment variable for installation of a package?

### DISCUSSION THREAD

**Discussion by g-kartik** [pdm-project/pdm#3042] [Q&A] [answered]
For example, for installation of Datumaro package with opencv-headless, we run.
DATUMARO_HEADLESS=1 pip install datumaro --no-binary=datumaro

What shall the equivalent of the above in pdm?

**Comment by frostming** [comment, marked as answer]
```
DATUMARO_HEADLESS=1 PDM_NO_BINARY=datumaro pdm add datumaro
```
